### PR TITLE
refine plot_[a|b|c]_xxxxx python files

### PR DESCRIPTION
**Pull request by xiaowuhu**
refine doc
Signed-off-by: xiaowuhu <hu.xiaowu@hotmail.com>

### CODE REVIEW

File: docs/tutorial/plot_bbegin_measure_time.py [hunk 1]
```diff
@@ -32,7 +32,10 @@
 from onnxruntime import InferenceSession
 from skl2onnx import to_onnx
 from skl2onnx.tutorial import measure_time
-
+import matplotlib
+# comment below line to show graph 
+matplotlib.use('Agg')
```
Comment (xadupre) [inline docs/tutorial/plot_bbegin_measure_time.py:37]: I'm wondering if this change affects the rendering in the documentation?

Comment (xiaowuhu) [reply docs/tutorial/plot_bbegin_measure_time.py:37]: please let me know where the document was generated? I'm going to do the testing.

Comment (xadupre) [reply docs/tutorial/plot_bbegin_measure_time.py:37]: You have to generate it yourself. The outcome is published in branch github-pages. Every modification in that branch is propagated into the website https://github.com/onnx/sklearn-onnx. You may use the following command line to build the documentation : 

```
python -m sphinx -j2 -v -N -T -b html -d build/doctrees docs dist/html
```

**Comment by xiaowuhu** [comment]
this was replaced by 2 other PRs

**Pull request closed without merging**